src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/CompletionSource.cs

```diff
@@ -28,10 +31,10 @@
 
 namespace Microsoft.CodeAnalysis.Editor.Implementation.IntelliSense.AsyncCompletion
 {
-    internal class CompletionSource : ForegroundThreadAffinitizedObject, IAsyncCompletionSource
+    internal class CompletionSource : ForegroundThreadAffinitizedObject, IAsyncExpandingCompletionSource
     {
         internal const string RoslynItem = nameof(RoslynItem);
-        internal const string TriggerSnapshot = nameof(TriggerSnapshot);
+        internal const string TriggerLocation = nameof(TriggerLocation);
```

Comment:
Should we use it from a session property not using own one in a property bag?

Reply:
Fixed

Reply:
Thank you! This is GREAT!

---
In reply to: [296052185](https://github.com/dotnet/roslyn/pull/36146#discussion_r296052185) [](ancestors = 296052185)